contrib/confluentinc/confluent-kafka-go/kafka/kafka.go

```diff
@@ -98,6 +98,13 @@ func (c *Consumer) startSpan(msg *kafka.Message) ddtrace.Span {
 		tracer.Tag("offset", msg.TopicPartition.Offset),
 		tracer.Measured(),
 	}
+
+	if c.cfg.tagFns != nil {
+		for key, tagFn := range c.cfg.tagFns {
+			opts = append(opts, tracer.Tag(key, tagFn(msg)))
+		}
+	}
+
```

Comment:
```suggestion
	if c.cfg.tagFns != nil {
		for key, tagFn := range c.cfg.tagFns {
			opts = append(opts, tracer.Tag(key, tagFn(msg)))
		}
	}
```
Please see point 3 in https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines. Happy to discuss these guidelines at any time and happy to change them if we see any issues.